### PR TITLE
New issue template: Edit this page.

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Creates a new issue template to take into account the case where a documentation page doesn't have a public source file. 

### Motivation

https://github.com/DataDog/documentation/pull/6384/

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/edit_this_page.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: Edit this page
+about: Many thanks for your feedback but the page you are trying to edit is not available publicly. Please fill out the information below and Datadog Documentation team will take care of updating the corresponding documentation page.
```
Comment (cswatt) [inline .github/ISSUE_TEMPLATE/edit_this_page.md]:
```suggestion
about: Thanks for your feedback, but the page you are trying to edit is not available publicly. Fill out the information below, and Datadog's Documentation team will take care of updating the corresponding documentation page.
```

File: .github/ISSUE_TEMPLATE/edit_this_page.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+---
+
+**Documentation page to update**:
```
Comment (cswatt) [inline .github/ISSUE_TEMPLATE/edit_this_page.md]: are we asking for them to put the URL? let's ask explicitly for a URL

File: .github/ISSUE_TEMPLATE/edit_this_page.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+**Describtion of the update to make**:
```
Comment (cswatt) [inline .github/ISSUE_TEMPLATE/edit_this_page.md]:
```suggestion
**Description of the update**:
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Feature request
-about: Suggest an idea for the Datadog documentation. To request a product feature, please [file a ticket to Datadog support team directly](https://docs.datadoghq.com/help/).
+about: "Suggest an idea for the Datadog documentation. To request a product feature, please file a ticket to Datadog support team directly: https://docs.datadoghq.com/help/".
```
Comment (cswatt) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
about: "Suggest an idea for the Datadog documentation. To request a product feature, please file a ticket to Datadog's support team directly: https://docs.datadoghq.com/help/".
```

File: .github/ISSUE_TEMPLATE/edit_this_page.md (hunk 2 above)
Comment (l0k0ms) [reply .github/ISSUE_TEMPLATE/edit_this_page.md]:
```suggestion
**Documentation page URL to update**:
```

**Pull request closed without merging**